examples/wkt/java/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//java:rules.bzl", "java_proto_library")
+
+java_proto_library(
+    name = "google_well_known_protos",
+    protos = ["@com_github_google_protobuf//:well_known_protos"],
```

Comment:
This is a bit off topic for this PR but are there any plans to add a `java_proto_library` rule to the protobuf/BUILD file so everyone could just depend on `@com_github_google_protobuf//:google_well_known_protos` as a proto dependency instead of grabbing the files and adding a `java_proto_library` to all of the projects that use them?